### ISSUE TITLE
Question about the image segmentation (use GAT)

### ISSUE THREAD

**Issue by tanjia123456** [danielegrattarola/keras-gat#31]
Hello, every person's brain can be represented by a graph, and there are a total of 80 people's data(80 subjects). As far as I know, the original GAT is to divide the data set of nodes, but here I want to divide the data set according to the subject. May I divide the first 70 subjects into the training set and the last 10 into the test set, so how do I use GAT for training? I want to use GAT for image segmentation.
Please take the time to guide me the following, it is greatly appreciated.
